packages/bsky/src/api/app/bsky/feed/getSuggestedFeeds.ts

```diff
@@ -19,7 +19,11 @@ export default function (server: Server, ctx: AppContext) {
         feedsRes.map((r) => r.uri),
         viewer,
       )
-      const genList = Object.values(genInfos)
+      const genList = Object.values(genInfos).sort((a, b) => {
```

Comment:
what about ordering this just based on `feedsRes`?

something like:
```ts
const genList = feedsRes.map(r => genInfos[r.uri])
```

Reply:
Ah yeah I shouldn't rely entirely on TS type huh. Copy that, fixing 👍